pvactools/tools/pvacseq/generate_aggregated_report.py

```diff
@@ -19,17 +19,28 @@ def define_parser():
         "output_file",
         help="The file path to write the aggregated report tsv to"
     )
+    parser.add_argument(
+        "--tumor-purity",
+        help="Value between 0 and 1 indicating the fraction of tumor cells in the tumor sample. Information is used for a simple estimation whether variants are subclonal or clonal based on VAF.",
```

Comment:
Suggested change:
help="Value between 0 and 1 indicating the fraction of tumor cells in the tumor sample. Information is used during aggregate report creation for a simple estimation of whether variants are subclonal or clonal based on VAF. If not provided, purity is estimated directly from the VAFs"